var/spack/repos/builtin/packages/py-gosam/package.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyGosam(PythonPackage):
+    """The package GoSam allows for the automated calculation of
+       one-loop amplitudes for multi-particle processes in renormalizable
+       quantum field theories."""
+
+    homepage = "https://gosam.hepforge.org"
+    url      = "https://gosam.hepforge.org/downloads/?f=gosam-2.0.4-6d9f1cba.tar.gz"
+
+    version('2.0.4', sha256='faf621c70f66d9dffc16ac5cce66258067f39f686d722a4867eeb759fcde4f44')
+    patch('gosam-1.2.4.patch')
```

Comment:
This patch seems to add Python 3 support, but you restrict things to Python 2.7. Is this patch actually useful then?

Reply:
I was trying to add Python3 support, but the task turned out too complicated. I will remove the patch for now.

Reply:
For now = until the developers make a PY3-compatible version